R/rank_list.R

```diff
@@ -152,11 +154,21 @@ dropNulls <- function(x) {
 #'   )
 #'   shiny::runApp(app)
 #' }
-update_rank_list <- function(css_id, text = NULL,
+
+update_rank_list <- function(css_id,
+                             text = NULL,
+                             labels = NULL,
                              session = shiny::getDefaultReactiveDomain()) {
-  inputId <- paste0("rank-list-", css_id)
-  message <- dropNulls(list(id = inputId, text = text))
-  session$sendInputMessage(inputId, message)
+  id <- session$ns(css_id)
```

Comment:
I'm for this change, as long as **all** other update methods and UI methods do this. (And is documented behavior) 

Otherwise, let's stay consistent and remove this.

Reply:
Thanks for the review @schloerke! Ok, Ill remove this